stripe/src/main/java/com/stripe/android/view/PaymentUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.stripe.android.R;
+
+import java.text.DecimalFormatSymbols;
+import java.text.NumberFormat;
+import java.util.Currency;
+import java.util.Locale;
+
+public class PaymentUtils {
+
+    /**
+     * Formats a monetary amount into a human friendly string.
+     */
+    static String getPriceString(@NonNull Context context, double amount, @NonNull Currency currency) {
+        if (amount == 0) {
+            return context.getResources().getString(R.string.shipping_free);
```

Comment:
If you pass in the "optional free string" as a parameter, this can be used elsewhere (and if there is no "free string" and the price is zero (or zero-ish), you can just display 0.00)